src/libcore/iter/traits.rs

```diff
@@ -352,6 +357,13 @@ pub trait Extend<A> {
     fn extend<T: IntoIterator<Item=A>>(&mut self, iter: T);
 }
 
+#[stable(feature = "rust1", since = "1.26.0")]
+impl <'a, A, E: Extend<A>> Extend<A> for &'a mut E {
```

Comment:
Can we really do this?  Regardless of crater, [RFC 1023](https://github.com/rust-lang/rfcs/blob/master/text/1023-rebalancing-coherence.md) calls `&mut T` a fundamental type, "where implementing a blanket impl [...] is a breaking change."

Reply:
@cuviper yeah I do believe it's a breaking change technically, but if it doesn't actually break anything in practice we *may* have leeway to land it.